### ISSUE TITLE
Hardware enhancement - indicator LEDs

### ISSUE THREAD

**Issue by aeksco** [aeksco/lyrebird#6]
I think the next revision of our board should have at least one LED that can be used to indicate any of the following:
- Hardware is powered on
- Hardware is linked to a device
- Data received from device
- Firmware updated successfully

We can use different blinking patterns for different indications.

An indicator LED would also be helpful as it draws a user's eye to the device to reduce the possibility that it's accidentally left behind somewhere.

@AaronPerl let me know your thoughts when you've got a chance

**Comment by aeksco** [comment]
Alternatively we could go with a more expensive RGB LED and use different colors to indicate what 'mode' the hardware is currently in (i.e. keyboard, mouse, remote, gamepad, etc.).

**Comment by AaronPerl** [comment]
I like the idea, though with an RGB LED I would prefer to use the different colors to indicate statuses (disconnected, connected but not in use, in use, etc.). We could even use a 2 in 1 red/green LED and use yellow for connected but not in use (such as http://www.digikey.com/product-detail/en/lite-on-inc/LTST-C155GEKT/160-1172-1-ND/269244).

Indicating a successful firmware update would be difficult due to us not being able to change the bootloader code, but we could probably do something with storing the firmware version in EEPROM and comparing it on startup to see if an update happened.

**Comment by aeksco** [comment]
Yeah the on-successful-firmware-update indication isn't really necessary, just spitballing. Not worth the development time to implement what is a very minor feature.

Definitely like using different colors to indicate different statuses - the 2-in-1 red/green LED is nice, and not too expensive in the quantities we're looking at for our prototype boards.
